### PR TITLE
Added kibana 6.4.2

### PR DESCRIPTION
**Pull request by TheKatarGit**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 1.6.3 (12 October, 2018)
+  - Add version 6.4.2 of Kibana;
+
```
Comment (anuriq) [inline CHANGELOG.md:3]: changelog is updated after merge, right before release

File: libraries/helpers.rb [hunk 2]
```diff
@@ -52,556 +52,572 @@ def kibana_artifact_checksum(version, install_method)
...
+        'darwin' => {
+          'x86_64' => {
+            'checksum' => '1078c4c5ced83459726522a268510e1c99d5e5f4fef7284c3b39ba8ca9c59bf2'
+          }
+        },
         'release' => {
           'x86_64' => {
-            'checksum' => '6c6fa47450cb05b96b97166a627d07c76591909da82d3bca0708bdebb889bf86',
-          },
+            'checksum' => 'f3a69709171ea8e28da754578ec519f33c423d3e3cbed6e81381f14cb8517383'
```
Comment (anuriq) [inline libraries/helpers.rb]: why are you doing this?  this contradicts style guide and checks have failed in CI. please, don't do this

**Review by anuriq — CHANGES_REQUESTED** [review]
please, revert back changes with comma

File: libraries/helpers.rb (hunk 2 above)
Comment (TheKatarGit) [reply libraries/helpers.rb]: @anuriq I ran rubocop before uploading this. Will remove shortly

**Comment by TheKatarGit** [comment]
@anuriq Done

**Pull request merged**